src/main/java/org/cryptomator/frontend/fuse/ReadWriteFileHandler.java

```diff
@@ -47,7 +48,10 @@ public ReadWriteFileHandler(OpenFileFactory openFiles, FileAttributesUtil attrUt
 	@Override
 	public int getattr(Path node, BasicFileAttributes attrs, FileStat stat) {
 		int result = super.getattr(node, attrs, stat);
-		if (result == 0 && supportsPosixFileAttributeView) {
+		if (result == 0 && attrs instanceof PosixFileAttributes) {
+			long mode = attrUtil.posixPermissionsToOctalMode(((PosixFileAttributes) attrs).permissions());
+			stat.st_mode.set(FileStat.S_IFREG | mode);
+		} else if (result == 0 && supportsPosixFileAttributeView) {
```

Comment:
isn't this a dead branch, since super.getattr always returns PosixFileAttributes if supported?

Reply:
also means that the `supportsPosixFileAttributeView` field can be removed from this class.